test/event/bond/verify-dao-bond-configuration-events.ts

```diff
@@ -7,7 +7,7 @@ import {
     daoMetaDataUpdateEvents,
     daoTreasuryUpdateEventLogs,
     daoTreasuryUpdateEvents
-} from './dao-bond-configuration-events'
+} from './dao-configuration-events'
```

Comment:
@CjHare should these files now be moved out into another `dao` directory since they are no longer specific to bonds

Reply:
Yeah, probably.

A new directory `dao` or `dao-configuration` would be the prime candidates.